examples/web/cmd/main.go

```diff
@@ -19,7 +20,7 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixNano())
 
-	autometrics.Init(nil, autometrics.DefBuckets)
+	amImpl.Init(nil, autometrics.DefBuckets)
```

Comment:
Fwiw, I think it's much clearer if this stays as `autometrics`. Whats the reason for the change?

Reply:
Actually, we cannot reexport everything, because even if we do

```go
package otel

type Context autometrics.Context
```
then the literal that gets generated would still need to use `autometrics.AlertConfiguration` and couldn't use `otel.AlertConfiguration`.

So we either have to use 2 imports in the end package, or duplicate all the code related to the context structure, and remember to keep those in sync whenever we make changes (as there won't be any compilation warning if the structures drift, and will only break later when users use the generator)